qualtran/simulation/quimb_sim.py

```diff
@@ -120,6 +124,18 @@ def _assign_outgoing(cxn: Connection) -> Soquet:
         if pos is not None:
             fix[tuple([binst])] = pos[binst]
 
+    inds = get_right_and_left_inds(cbloq.signature)
+    if len(inds) != 2:
+        return tn, fix
```

Comment:
add a quick comment that this is for the case where the tensor network is a vector (bra or ket) instead of an operation (unitary)

Reply:
done